GpsDataCapturer/wear/src/main/java/com/google/gpsdatacapturer/GpsMainActivity.java

```diff
@@ -0,0 +1,169 @@
+package com.google.gpsdatacapturer;
+
+import android.Manifest;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.content.ServiceConnection;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.os.IBinder;
+import android.support.wearable.activity.WearableActivity;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+
+import androidx.core.app.ActivityCompat;
+
+import com.google.sharedlibrary.GpsDataCaptureService;
+import com.google.sharedlibrary.GpsDataCaptureService.GpsDataCaptureBinder;
+
+public class GpsMainActivity extends WearableActivity {
+    private static final String TAG = "GpsMainActivity";
+    private static GpsDataCaptureService gpsDataCaptureService;
+    private static Intent serviceIntent;
+    private static boolean isBound = false;
+    private static final int REQUEST_CODE = 1340;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_gps_main);
+
```

Comment:
Can you also add a check if location is enabled

Reply:
Will do.